terraform/eks.tf

```diff
@@ -115,7 +124,7 @@ resource aws_eks_node_group "eks_nodes" {
   node_role_arn   = aws_iam_role.iam_for_eks_nodes.arn
   subnet_ids      = ["${aws_subnet.eks_subnet1.id}", "${aws_subnet.eks_subnet2.id}"]
 
-  instance_types  = ["t3.nano"]
+  instance_types  = ["t2.medium"]
```

Comment:
why not nano?

Reply:
maybe you should add a tflint to this repo

Reply:
@jonjozwiak reached me and sent me this good explanation: 
```
EKS uses IPs for each of the pods running in the cluster.  A t3.nano can only have 4 IPs.  One of these is the host.  In a single node cluster the other 3 would be kube-proxy, coredns, and aws-node.  So you would have no room to run any workloads.
```